feat(changelog): secret-manager-added-secret-manager-is-generally 2023-04-28 (#1606)
scaleway/docs-content: changelog/april2023/2023-04-28-secret-manager-added-secret-manager-is-generally.mdx

---
title: Secret Manager is now in General Availability
status: added
author:
    fullname: 'Join the #secret-manager channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-04-28
category: security-identity
product: secret-manager
---

Secret Manager is now in General Availability.

For more information, refer to the [Secret Manager product documentation](/identity-and-access-management/secret-manager/quickstart/) or the [Secret Manager product website](https://www.scaleway.com/en/secret-manager/)

> 

Reviewer: ```suggestion
```

---

> title: Secret Manager is generally available (FA)

Reviewer: ```suggestion
title: Secret Manager is generally available (GA)
```

status: added
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #secret-manager channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-04-28
category: security-identity
product: secret-manager
---

> GA Secret Manager available now! 

Reviewer: ```suggestion
Secret Manager is generally available now! 
```

> For more information, see https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/quickstart/

Reviewer: ```suggestion
For more information, refer to the [Secret Manager product documentation](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/quickstart/).
```

> website https://www.scaleway.com/fr/secret-manager/

Reviewer: ```suggestion
[Secret Manager product website](https://www.scaleway.com/en/secret-manager/).
```

> website https://www.scaleway.com/fr/secret-manager/

Reviewer 2: ```suggestion
```
